docs/endaq/ide_usage.rst

```diff
@@ -0,0 +1,685 @@
+          <td id="T_d2cbb_row12_col4" class="data row12 col4" >00:00.0979</td>
+          <td id="T_d2cbb_row12_col5" class="data row12 col5" >00:18.0910</td>
+          <td id="T_d2cbb_row12_col6" class="data row12 col6" >00:17.0931</td>
+          <td id="T_d2cbb_row12_col7" class="data row12 col7" >180</td>
+          <td id="T_d2cbb_row12_col8" class="data row12 col8" >10.04 Hz</td>
+        </tr>
+        <tr>
+          <th id="T_d2cbb_level0_row13" class="row_heading level0 row13" >13</th>
+          <td id="T_d2cbb_row13_col0" class="data row13 col0" >59.1</td>
+          <td id="T_d2cbb_row13_col1" class="data row13 col1" >Control Pad Temperature</td>
+          <td id="T_d2cbb_row13_col2" class="data row13 col2" >Temperature</td>
+          <td id="T_d2cbb_row13_col3" class="data row13 col3" >°C</td>
+          <td id="T_d2cbb_row13_col4" class="data row13 col4" >00:00.0979</td>
+          <td id="T_d2cbb_row13_col5" class="data row13 col5" >00:18.0910</td>
+          <td id="T_d2cbb_row13_col6" class="data row13 col6" >00:17.0931</td>
+          <td id="T_d2cbb_row13_col7" class="data row13 col7" >180</td>
+          <td id="T_d2cbb_row13_col8" class="data row13 col8" >10.04 Hz</td>
+        </tr>
+        <tr>
+          <th id="T_d2cbb_level0_row14" class="row_heading level0 row14" >14</th>
+          <td id="T_d2cbb_row14_col0" class="data row14 col0" >76.0</td>
+          <td id="T_d2cbb_row14_col1" class="data row14 col1" >Lux</td>
+          <td id="T_d2cbb_row14_col2" class="data row14 col2" >Light</td>
+          <td id="T_d2cbb_row14_col3" class="data row14 col3" >Ill</td>
+          <td id="T_d2cbb_row14_col4" class="data row14 col4" >00:00.0000</td>
+          <td id="T_d2cbb_row14_col5" class="data row14 col5" >00:18.0737</td>
+          <td id="T_d2cbb_row14_col6" class="data row14 col6" >00:18.0737</td>
+          <td id="T_d2cbb_row14_col7" class="data row14 col7" >71</td>
+          <td id="T_d2cbb_row14_col8" class="data row14 col8" >3.79 Hz</td>
+        </tr>
+        <tr>
+          <th id="T_d2cbb_level0_row15" class="row_heading level0 row15" >15</th>
+          <td id="T_d2cbb_row15_col0" class="data row15 col0" >76.1</td>
+          <td id="T_d2cbb_row15_col1" class="data row15 col1" >UV</td>
+          <td id="T_d2cbb_row15_col2" class="data row15 col2" >Light</td>
+          <td id="T_d2cbb_row15_col3" class="data row15 col3" >Index</td>
+          <td id="T_d2cbb_row15_col4" class="data row15 col4" >00:00.0000</td>
+          <td id="T_d2cbb_row15_col5" class="data row15 col5" >00:18.0737</td>
+          <td id="T_d2cbb_row15_col6" class="data row15 col6" >00:18.0737</td>
+          <td id="T_d2cbb_row15_col7" class="data row15 col7" >71</td>
+          <td id="T_d2cbb_row15_col8" class="data row15 col8" >3.79 Hz</td>
+        </tr>
+      </tbody>
+    </table>
+
+
+
+
+The results can be filtered by :doc:`measurement type <ide_measurement>`:
+
```

Comment:
so while the table looks really nice here, I don't necessarily like the lack of visual distinction between the table's formatting and the pain text formatting. E.g. to my eyes, the last line "The results can be filtered by measurement type:" bleeds together with the table above it:

![image](https://user-images.githubusercontent.com/5862803/146598611-93c74f47-7343-4b46-b73a-0d219fc62c58.png)

I think instead I would prefer the entire thing to be a `pycon` block, so that the command *and* it's output are both stylistically wrapped in a code-style block. (I've seen this done [in the pandas docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.resample.html) quite a bit, though the tables tend to be smaller.)
```suggestion
.. code:: pycon

    >>> get_channel_table(doc)
    ...

The results can be filtered by :doc:`measurement type <ide_measurement>`:

```

Reply:
The more I think about this one specifically, the more I think we could do it later in another PR. so if you want to act on it here I'd say it's fine; I can make an issue for it and we can deliberate over it later.

Reply:
I do want to make this happen, but it might take a little more work. I'd rather get this PR merged sooner than later, so another PR would be better.